src/util/states.ts

```diff
@@ -427,19 +427,29 @@ function shouldSilent(el: Element, e: ElementEvent) {
 
 export function allLeaveBlur(api: ExtensionAPI) {
     const model = api.getModel();
+    const leaveBlurredSeries: SeriesModel[] = [];
+    const allComponents: ComponentModel[] = [];
     model.eachComponent(function (componentType, componentModel) {
         const componentStates = getComponentStates(componentModel);
+        const isSeries = componentType === 'series';
+        const view = isSeries ? api.getViewOfSeriesModel(componentModel as SeriesModel)
+            : api.getViewOfComponentModel(componentModel);
+        !isSeries && allComponents.push(componentModel);
```

Comment:
Seems we can save view instead of model here directly and there is no need to use `getViewOfComponentModel` in the following code again.